docs/source/gateway/index.rst

```diff
@@ -1,6 +1,951 @@
+                },
+                "metadata": {"finish_reason": "stop"},
+            }
+        ],
+        "metadata": {
+            "input_tokens": 13,
+            "output_tokens": 15,
+            "total_tokens": 28,
+            "model": "gpt-3.5-turbo-0301",
+            "route_type": "llm/v1/chat",
+        },
+    }
+
+
+Step 7: Send Requests Using the Client API
+------------------------------------------
+See the :ref:`gateway_client_api` section for further information.
+
+Step 8: Send Requests to Routes via REST API
+--------------------------------------------
+You can now send requests to the exposed routes.
+See the :ref:`REST examples <gateway_rest_api>` for guidance on request formatting.
+
+Step 9: Compare Provider Models
+-------------------------------
+Here's an example of adding a new model from a provider to determine which model instance is better for a given use case.
+
+Firstly, update the :ref:`MLflow AI Gateway config <gateway_configuration>` YAML file with the additional route definition to test:
+
+.. code-block:: yaml
+
+    routes:
+      - name: completions
+        type: llm/v1/completions
+        model:
+          provider: openai
+          name: gpt-3.5-turbo
+          config:
+            openai_api_base: https://api.openai.com/v1
+            openai_api_key: $OPENAI_API_KEY
+      - name: completions-gpt4
+        type: llm/v1/completions
+        model:
+          provider: openai
+          name: gpt-4
+          config:
+            openai_api_base: https://api.openai.com/v1
+            openai_api_key: $OPENAI_API_KEY
+
+This updated configuration adds a new completions route ``completions-gpt4`` while still preserving the original ``completions``
+route that was configured with the ``gpt-3.5-turbo``  model.
+
+Once the configuration file is updated, simply save your changes. The Gateway will automatically create the new route with zero downtime.
+
+At this point, you may use the :ref:`gateway_fluent_api` to query both routes with similar prompts to decide which model performs best for your use case.
+
+If you no longer need a route, you can delete it from the configuration YAML and save your changes. The AI Gateway will automatically remove the route.
+
+.. _gateway-concepts:
+
+Concepts
+========
+
+There are several concepts that are referred to within the MLflow AI Gateway APIs, the configuration definitions, examples, and documentation.
+Becoming familiar with these terms will help in configuring new endpoints (routes) and ease the use of the interface APIs for the AI Gateway.
+
+.. _providers:
+
+Providers
+---------
+The MLflow AI Gateway is designed to support a variety of model providers.
+A provider represents the source of the machine learning models, such as OpenAI, Anthropic, and so on.
+Each provider has its specific characteristics and configurations that are encapsulated within the model part of a route in the MLflow AI Gateway.
+
+Supported Provider Models
```

Comment:
Not a blocker, but a matrix (column: route, row: provider) might be easier to understand.